src/request.js

```diff
@@ -234,7 +238,7 @@ export function getNodeRequestOptions(request) {
 
 	// HTTP-network-or-cache fetch step 2.11
 	if (!headers.has('User-Agent')) {
-		headers.set('User-Agent', 'node-fetch/1.0 (+https://github.com/bitinn/node-fetch)');
+		headers.set('User-Agent', `${packageName}/${version} [Node ${process.version}] (+${homepage})`);
```

Comment:
since `${version}`  is just a number. I think you should do `v${version}`

Reply:
'v' prefix is traditionally omitted in userAgent strings.